frontend/src/app/modules/user-detail/components/input/input.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+
+@Component({
+  selector: 'app-input',
```

Comment:
please rename to `app-user-input` to avoid name conflict

Reply:
changed